src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
```

Comment:
It's not recommended to use `any` as a type in TypeScript. It's a good practice to create an interface that describes the shape of the context. This way, you can take full advantage of TypeScript's type checking.